ui/jasmine_test/spec/helpers/stubs/StubContext.js

```diff
@@ -0,0 +1,13 @@
+import {StubSource} from './StubSource.js';
+import {StubAnalyser} from './StubAnalyser.js';
+
+export class StubContext {
```

Comment:
Can you name this class `StubAudioContext` to reduce confusion, as there is also a canvas context floating around in this code